source/slang/slang-emit-c-like.cpp

```diff
@@ -3537,6 +3537,10 @@ void CLikeSourceEmitter::emitStructDeclarationsBlock(IRStructType* structType, b
                 emitPackOffsetModifier(fieldKey, fieldType, packOffsetDecoration);
             }
         }
+        if(this->getTargetProgram()->getOptionSet().getBoolOption(CompilerOptionName::AllowGLSL))
```

Comment:
Let's try to avoid wrapping things in `if (CompilerOptionName::AllowGLSL)`, because I want to remove this option going forward.